metaflow/linters/ruff/ruff.toml

```diff
@@ -0,0 +1,18 @@
+# refer more at: https://docs.astral.sh/ruff/settings/#format_indent-style
```

Comment:
@savingoyal I request better strategy/line-of-action to infer the ruff settings a.k.a `ruff.toml` at client's end, in the form a global, project-level, or override-able on demand setting. One way to support is 
```
python <flow>.py --lint-config global|<file>|<dict>
```
thoughts?